### DISCUSSION TITLE
"NOT" or "AND NOT" do not working

### DISCUSSION THREAD

**Discussion by ishaisagi** [microsoft-search/pnp-modern-search#3249] [Q&A]
**Version used**
4.6.1

**Describe the bug**
boolean operator **NOT** does not work. 
For example, searching for "benefits NOT schedule" (without the quotes!) works will with out-of-the-box search, but with PNP returns items that have both terms.
When I change the "NOT" to "AND", I get less results - so the "AND" seems to work (finds less results that have both terms in them).
Changing the NOT to "OR" gives me a lot more results - as expected.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. add a search box wp
2. add a search results wp
3. configure results to connect to the search box
4. configure results to only return results from a specific content type of a list item that is used in the site and is indexed(not sure if this is related!) in the query template:
```
{searchTerms} 
AND ContentTypeId:0x010081DC5A7E6208024FA7FA528CBC88501706*
```
also tried:
`{searchTerms} ContentTypeId:0x010081DC5A7E6208024FA7FA528CBC88501706*`
5. create three list items - one with the word "modern", one with "modern search" and one with "search" in the body field
6. perform a search using "Modern NOT search"

**Expected behavior**
It should only result with one item - the one saying "modern"

**Additional context**
I found a related issue from 2021 that is supposed to have resolved this bug, but it seems the version I am on (cannot upgrade at the moment) has the same issue.
https://github.com/microsoft-search/pnp-modern-search/issues/1136

**Comment by wobba** [comment]
We cannot really control boolean parsing on the API from the web part. If the expression is correct on the http request, then this is an issue with the API and should be logged accordingly with support.
